src/app/main/component/events/components/events-list/events-list.component.ts

```diff
@@ -163,10 +164,6 @@ export class EventsListComponent implements OnInit, OnDestroy {
     this.searchToggle = !this.searchToggle;
   }
 
-  public showFavourite(): void {
```

Comment:
as i see we use this function in HTML file, why you deleted it?

Reply:
it was my function which i decided to delete because it is not finished

Reply:
Maybe it makes sense to delete it also in the template?